docs/tdvt.md

```diff
@@ -120,19 +120,11 @@ TDVT uses the 'Calcs' and 'Staples' tables.
 
    ![]({{ site.baseurl }}/assets/tdvt_connection_2.png)
 
-1. Open the TDS file in a text editor and embed the password in the <connection> tag of the TDS file next to the existing 'username' value.
-   Save the file.
+1. Run `tdvt action --add_ds mydb`. Chose to generate the password file and pick the logical query config. This will create a mydb.ini file under /config and will modify your two TDS files to rename the connection and link them to the tds/mydb.password password file.
 
-1. Repeat this for the 'Staples' table.
+1. Edit the generated tds/mydb.password file and enter the password for your connection. This can also be done manually. See [The Sample TDS Files](https://github.com/tableau/connector-plugin-sdk/tree/master/tdvt/samples/tds)
```

Comment:
```suggestion
1. Edit the generated tds/mydb.password file and enter the password for your connection. 
This can also be done manually by creating a new file named using the format `datasource.password` (e.g. `postgres.password`). Save your password in the file using the following key/value format `datasourcetestname;password`, e.g.:
```
 postgrestestname;hunter2
```
See [The Sample TDS Files](https://github.com/tableau/connector-plugin-sdk/tree/master/tdvt/samples/tds)
```